src/crimson/osd/osd_operations/osdop_params.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include "messages/MOSDOp.h"
+
+struct OSDOpParams {
+  const MOSDOp& req;
+  eversion_t at_version;
+  eversion_t pg_trim_to;
+  eversion_t min_last_complete_ondisk;
+  eversion_t last_complete;
+
+  OSDOpParams(const MOSDOp& op, eversion_t ver, eversion_t ptt,
```

Comment:
nit, could use the same name as the member variables. like
```c++
OSDOpParams(const MOSDOp& req, 
            eversion_t at_version,
            eversion_t pg_trim_to, /**/) :
  req{req}, at_version{at_version}, pg_trim_to{pg_trim_to} // ...
{}
```
so one does not need to map `ptt` to `pg_trim_to` when reading it.